docs-new/docs/graphql/core/getting-started/docker-simple.mdx

```diff
@@ -62,25 +64,24 @@ Head to `http://localhost:8080/console` to open the Hasura console.
 
 Navigate to `Data -> Manage -> Connect Database`:
 
-<div class="thumbnail" alt="Connect database" width="1000px">
+<Thumbnail
+  src="/img/graphql/cloud/getting-started/connect-db-console.png"
+  alt="Connect database"
+  width="1000px"
+/>
 
-/img/graphql/cloud/getting-started/connect-db-console.png
-
-</div>
-
-Enter your database connection URL. *(See the note below if you do not
-have an existing database)*
+Enter your database connection URL. _(See the note below if you do not
+have an existing database)_
 
 Click `Connect Database`.
 
-<div class="thumbnail" alt="Enter URL for existing database"
-width="700px">
-
-/img/graphql/core/getting-started/connect-db.png
+<Thumbnail
+  src="/img/graphql/core/getting-started/connect-db.png"
+  alt="Enter URL for existing database"
+  width="700px"
+/>
 
-</div>
-
-<div class="admonition">
+:::info Note
 
 Starting from scratch
```

Comment:
`Starting from scratch` is the correct title, pls fix and always cross check with live for content and UI correctness.
`:::info Starting from scratch`